spec/features/groups_edit_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'when opening the group\'s edit page', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Tom', email: 'tom@example.com', password: 'topsecret')
+    @user.confirm
```

Comment:
- [ ] You also need to remove all the `@user.confirm` lines in all the spec files contained in the `spec/features` folder. After that, most of these features' specs still pass, but you still need to take a look at the failing ones and fix them.

  ![image](https://github.com/katarighe/budget-app/assets/81468675/de2bd093-b8f6-44c7-8b0f-3347ea01a388)

Reply:
✅ Solved